public/src/js/widgets/columns/packages.js

```diff
@@ -23,17 +23,37 @@ export default class Package extends ColumnWidget {
         this.displayName = ko.observable();
         this.searchResults = ko.observableArray();
         this.searchedPackages = ko.observable();
+        this.editingPackage = ko.observable(null);
 
         this[bouncedSearch] = debounce(performSearch.bind(this), CONST.searchDebounceMs);
+
+        this.listenOn(mediator, 'package:edit', (packageId) => {
```

Comment:
the body of the callback shouldn't be defined in the constructor and instead of sending a package id, you could send the whole package object maybe